pkgs/servers/tautulli/default.nix

```
@@ -1,54 +1,48 @@
-{stdenv, fetchFromGitHub, python }:
+{ lib, fetchFromGitHub, python, buildPythonApplication, bash, setuptools, wrapPython, makeWrapper }:
 
-stdenv.mkDerivation rec {
-  version = "2.2.4";
+buildPythonApplication rec {
   pname = "Tautulli";
+  version = "2.6.1";
+  format = "other";
 
-  pythonPath = [ python.pkgs.setuptools ];
-  buildInputs = [ python.pkgs.setuptools ];
-  nativeBuildInputs = [ python.pkgs.wrapPython ];
+  pythonPath = [ setuptools ];
+  nativeBuildInputs = [ wrapPython makeWrapper ];
 
   src = fetchFromGitHub {
     owner = "Tautulli";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0yg7r7yscx6jbs1lnl9nbax3v9r6ppvhr4igdm3gbvd2803j8fs7";
+    sha256 = "QHpVIOtGFzNqAEcBCv48YWO4pYatbTe/CWwcwjbj+34=";
   };
 
-  buildPhase = ":";
+  doBuild = false;
 
   installPhase = ''
-    mkdir -p $out
-    cp -R * $out/
+    mkdir -p $out/{bin,libexec/tautulli}
```

```suggestion
    mkdir -p $out/bin $out/libexec/tautulli
```

Small nitpick to increase readability.